Backend.Core/ItemsManagers/LayersManager.cs

```diff
@@ -2,33 +2,65 @@
 using Backend.Models.RoomItems;
 using System.Collections.Generic;
 using System.Linq;
+using Serilog;
 namespace Backend.Core
 {
     internal class LayersManager : IRoomItemsManager
     {
-        private readonly Dictionary<string, Layer> _layers = new Dictionary<string, Layer>();
+        //private readonly Dictionary<string, Layer> _layers = new Dictionary<string, Layer>();
+        private List<Layer> _layers = new List<Layer>();
         public int Count => _layers.Count;
-        private List<Layer> LayerList { get => _layers.Values.ToList(); }
+        private int layerNameCount = 1;
```

Comment:
Please use property `Count `or change it to:
`private int NextLayerName get =>  $"Layer {Count+1}";`

For the future please follow the convention that private member variables are writen with _underscore ;)